### PR TITLE
NET-129: Turn Signal Actions

### PR DESCRIPTION
**Pull request by abhishek9686**
## Describe your changes
-> Peer signal actions 
## Provide testing steps
refer https://github.com/gravitl/netclient/pull/398
## Checklist before requesting a review
- [x] My changes affect only 10 files or less.
- [x] I have performed a self-review of my code and tested it.
- [ ] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [ ] If it is a bugfix, my code is <= 200 lines.
- [ ] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [x] Netmaker is awesome.

### CODE REVIEW

File: models/host.go [hunk 1]
```diff
@@ -121,6 +121,16 @@ const (
 	UpdateKeys = "UPDATE_KEYS"
 )
 
+// SignalAction - turn peer signal action
+type SignalAction string
+
+const (
+	// DissolveConn - action to stop using turn connection
+	DissolveConn SignalAction = "DISSOLVE_CONNECTION"
```
Comment (TobiaszCudnik) [inline models/host.go:129]: Is it the same as "disconnect"?

**Review by TobiaszCudnik — APPROVED** [review]

File: models/host.go (hunk 1 above)
Comment (abhishek9686) [reply models/host.go:129]: yes

Comment (TobiaszCudnik) [reply models/host.go:129]: So its better to call it that way

Comment (abhishek9686) [reply models/host.go]: done,makes sense

**Review by TobiaszCudnik — APPROVED** [review]

**Review by mattkasun — APPROVED** [review]

**Pull request merged**